### ISSUE TITLE
Add Bower Support

### ISSUE THREAD

**Issue by DavoBR** [Morgul/ui-bootstrap4#11]

**Comment by Morgul** [comment]
Here's some good news/bad news.

### Bad News

Bower is not _officially_ supported by `ui-bootstrap4`. Since it no longer recommends being used for new projects, and I have limited resources, I have no intention of maintaining support for it.

### Good News

It _should_ just work. With some minor caveats.

```bash
$ bower install morgul/ui-bootstrap4
```
However, there's no `dist` folder. Instead, the files you're looking for are under `bower_components/ui-bootstrap4/docs`.  That folder contains a copy of every version, ever. Needless to say, it's a little overkill, but it'll work.

If you (or anyone else) wants to improve that situation and make it less bloatedr, I will absolutely take merge requests.

**Comment by Zelarith** [comment]
Hi, 
I'm also working with bower.
I tried

`$ bower install ui-bootstrap4`

And it successfully added the directory to my bower.
The problem though is that my brunch-config.js doesn't seem to be able to pick it up, as it doesn't have any bower.json to indicate it where to pick the main file.

Anyone facing this issue with a handy workaround?

**Comment by Zelarith** [comment]
Hi, 
I'm also working with bower.
I tried

`$ bower install ui-bootstrap4`

And it successfully added the directory to my bower.
The problem though is that my brunch-config.js doesn't seem to be able to pick it up, as it doesn't have any bower.json to indicate it where to pick the main file.

Anyone facing this issue with a handy workaround?

**Comment by Morgul** [comment]
If this is really a need, I'll take a merge request to add `bower.json`.

**Comment by Zelarith** [comment]
That'd be really nice :)

While bower is deprecated it is still maintained, and I see a lot of projects still using it.

Of course, new projects should use Yarn over Bower, and I see how one could think an old project would be unlikely to come without ui-bootstrap integrated.

As a matter of fact, my project already had it, but the wrong version.
It had Bootstrap CSS 4.1.3, with UI-Bootstrap 2.0.5 (wich was built for Bootstrap CSS 3.3.7).
Some classes were updated between these 2 versions of Bootstrap CSS, and, as a result, we can't go back to Bootstrap CSS 3.3.7, but some of the modules of UI-Bootstrap aren't working properly either.

The only thing to do is to upgrade to UI-Bootstrap4, but we are stuck with bower...
Either that or we need to import it as a JS asset, wich is kinda ugly.

**Comment by Morgul** [comment]
@Zelarith Yeah, when I started this project I was in a similar boat, but we did the upgrade to a new build system (We've gone through three since the bower days, TBH)... and even though it's not hard, I do know what kind of push back you can get on those large changes to "legacy" products. I'm really, really trying to encourage people to use this the only way the maintainer does, but alas, such is the nature of this _particular_ project.

I need to just be a little bit more open, and let the people who use bower deal with issues with bower, if they come up. Kinda hate doing that, but it's better than a "no" in this case.

**Comment by Morgul** [comment]
Reopening this to track.
